homr/title_detection.py

```diff
@@ -1,32 +1,44 @@
 import re
 
-import easyocr
+from easyocr import Reader
 
 from homr.debug import Debug
 from homr.model import Staff
 
-reader = easyocr.Reader(["de", "en"], gpu=False, verbose=False)
 
+reader = Reader(["de", "en"], gpu=False, verbose=False)

```

Comment:
I wonder if we could try a change here:

If we define this global here as "reader: Reader | None = None" and then in detect_title if reader is None, we initialize the reader before we use it. By doing this, all the heavy initialization logic goes onto the thread we spawned and the import in main.py could just become an ordinary import.